src/nvim/memline.h

```diff
@@ -10,3 +10,16 @@
 
 /// LINEEMPTY() - return true if the line is empty
 #define LINEEMPTY(p) (*ml_get(p) == NUL)
+
+// Values for the flags argument of ml_delete_flags().
+enum {
+  ML_DEL_MESSAGE = 1,  // may give a "No lines in buffer" message
+  ML_DEL_UNDO = 2,     // called from undo, do not update textprops
```

Comment:
Not sure about this, but `ML_DEL_UNDO` actually doesn't do anything in Nvim...
```suggestion
  ML_DEL_UNDO = 2,     // called from undo
```

Reply:
Could comment it out or rename plus reassign to 0
or use `ml_delete()` with a comment for `ML_DEL_UNDO`
if no one else finds a use for it.